packages/devtools_app/lib/src/table.dart

```diff
@@ -740,6 +739,10 @@ class _TableState<T> extends State<_Table<T>> with AutoDisposeMixin {
         }
       });
     });
+
+    if (widget.activeSearchMatchNotifier != null) {
```

Comment:
redundant check for the notifier being not null.

Reply:
removed now that the check is inside _initSearchListener